app/src/main/java/com/sdp13epfl2021/projmag/VideoCv.kt

```diff
@@ -0,0 +1,47 @@
+package com.sdp13epfl2021.projmag
+
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
+import android.os.Bundle
+import android.provider.MediaStore
+import android.widget.Button
+import android.widget.MediaController
+import android.widget.VideoView
+import androidx.appcompat.app.AppCompatActivity
+
+
+class VideoCv : AppCompatActivity() {
```

Comment:
Maybe add a `submit` button+method which set the video URI as activity result and finishes the activity.
Add a key in a companion object, to retrieve the result from the activity which called the `VideoCv` activity